<launch>

	<node pkg="first_project" type = "odom_node" name = "odom_node" output="screen" />
	
	<!-- set time = simulation time of the bag, and not computer's (real) time -->
	<param name="/use_sim_time" value="true" />
	
	<!-- set the requested parameters -->
	<param name="starting_x" value="0" />
	<param name="starting_y" value="0" />
	<param name="starting_th" value="0" />
	
	<!-- static tf -->
	<!-- Assuming that lidars are at +-2 and +-1 meters from the vehicle center -->
	<node pkg="tf" type="static_transform_publisher" name="front_left" args="1.85 0.93 0.0 0.81 0.0 3.14 vehicle_centre sick_front_left 100" />
	<node pkg="tf" type="static_transform_publisher" name="front_right" args="1.85 -0.93 0.0 -0.76 0.0 3.14 vehicle_centre sick_front_right 100" />
	<node pkg="tf" type="static_transform_publisher" name="rear_left" args="-1.85 0.93 0.0 2.38 0.0 3.14 vehicle_centre sick_rear_left 100" />
	<node pkg="tf" type="static_transform_publisher" name="rear_right" args="-1.75 -0.8 0.0 -2.30 0.0 3.14 vehicle_centre sick_rear_right 100" />


	
</launch>

